### PR TITLE
Use actual generated d.ts from webcola project

### PR DESCRIPTION
**Pull request by nchen63**
webcola project is written in typescript

the file is from https://github.com/tgdwyer/WebCola/commit/78c3bf51a7305942b79dc012746ea1fb9fe90745

note that the original file is modified to remove dependency on d3.d.ts

### CODE REVIEW

File: webcola/webcola.d.ts [hunk 1]
```diff
@@ -1,71 +1,1097 @@
-// Type definitions for webcola 3.0.1
+// Type definitions for webcola 3.1.1
 // Project: https://github.com/tgdwyer/WebCola
-// Definitions by: Qinfeng Chen <https://github.com/qinfchen>
```
Comment (vvakame) [inline webcola/webcola.d.ts]: please keep original author. add your name to end of line.
like this https://github.com/borisyankov/DefinitelyTyped/blob/master/jquery/jquery.d.ts#L3

**Comment by vvakame** [comment]
*webcola/webcola.d.ts*

to author(@qinfchen). could you review this PR?
:+1: or :-1:?

check list

* [ ] pass the Travic-CI test?

**Comment by vvakame** [comment]
@tgdwyer can I merge this pull request?

**Comment by tgdwyer** [comment]
Yes, thanks!

**Comment by vvakame** [comment]
thanks mate!

**Pull request merged**